scripts/vm/hypervisor/xenserver/xenserver70/patch

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is the content of this same as that of XS6.5? If so then this is not needed, the 6.5 one can be reused.